examples/reference/elements/bokeh/Chord.ipynb

```diff
@@ -0,0 +1,98 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"contentcontainer med left\" style=\"margin-left: -50px;\">\n",
+    "<dl class=\"dl-horizontal\">\n",
+    "  <dt>Title</dt> <dd> Graph Element</dd>\n",
+    "  <dt>Dependencies</dt> <dd>Matplotlib</dd>\n",
+    "  <dt>Backends</dt> \n",
+    "    <dd><a href='./Chord.ipynb'>Bokeh</a></dd>\n",
+    "    <dd><a href='../matplotlib/Chord.ipynb'>Matplotlib</a></dd>\n",
+    "</dl>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import holoviews as hv\n",
+    "from bokeh.sampledata.les_mis import data\n",
+    "\n",
+    "hv.extension('bokeh')\n",
+    "%output size=200"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The ``Chord`` element allows representing the inter-relationships between data points in a graph. The nodes are arranged radially around a circle with the relationships between the data points drawn as arcs (or chords) connecting the nodes. The number of chords is scaled by a weight declared as a value dimension on the ``Chord`` element.\n",
+    "\n",
+    "If the weight values are integers, they define the number of chords to be drawn between the source and target nodes directly. If the weights are floating point values, they are normalized to a default of 500 chords, which are divided up among the edges. Any non-zero weight will be assigned at least one chord.\n",
+    "\n",
+    "The ``Chord`` element is a type of ``Graph`` element and shares the same constructor. The most basic constructor accepts a columnar dataset of the source and target nodes and an optional value. Here we supply a dataframe containing the number of dialogues between characters of the *Les Misérables* musical. The data contains ``source`` and ``target`` node indices and an associated ``value`` column:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "links = pd.DataFrame(data['links'])\n",
+    "print(links.head(3))\n",
```

Comment:
I would prefer to see this cell split:

```python
links = pd.DataFrame(data['links'])
links.head(3)
```

Then:

```python
hv.Chord(links)
```

I also notice the kernel metadata is still present in the notebooks. Could you please clear that out?

Reply:
Sure.